### DISCUSSION TITLE
Neofita. Gestione termosifoni e climatizzatori da remoto.

### DISCUSSION THREAD

**Discussion by danieva79** [indomus/forum#3252] [GENERALE - Discussioni generiche]
Ciao a tutti, mi chiamo Daniele, sono appena arrivato qui e piacere di conoscervi.
Sto ristrutturando una abitazione in un condominio per farne una casa vacanze e vorrei installare dei dispositivi che mi permettano di gestire da remoto alcune "funzioni" della casa. 
Ho acquistato (ma faccio ancora in tempo a renderli) le valvole TadoX: vorrei sapere se devo evitare di acquistare il bridge ma posso utilizzarli da remoto acquistando un amazon echo (o qualcosa di simile). In alternativa, considerando che il mio obiettivo è spegnere i termosifoni nelle varie stanze se non ci sono ospiti comodamente da remoto: che prodotto suggerite di acquistare ad uno che non ha particolari compentenze - ma tanta curiosità si - nel mondo dell'automazione?
Quanto ai climatizzatori, saranno dei "semplici" Hisense ma hanno una app che mi permetterà di gestirli da remoto. Quello che però vorrei è poterli spegnere quando, per esempio, non sia rilevata la presenza di un ospite in stanza per più di 30 minuti. Anzi, mi piacerebbe proprio che questo avvenisse in automatico. Che tipo di "sistema" dovrei acquistare per questo?

Grazie a chi volesse contribuire. ed ancora piacere

**Comment by Scialla** [comment]
https://help.tado.com/en/articles/9184565-how-can-i-set-up-my-tado-x-devices-with-alexa

Devi avere un Matter-compatible Alexa controller (non serve Thread)

Devi installare le skill Tado  e HiSense per Alexa
Devi poi provare a fare delle routine su Alexa per ottenere quello che chiedi

Se invece utilizzassi HomeAssistant (che noi consigliamo) potresti fare questo e molto altro

**Reply by KC-inDomus** [reply]
sì che serve thread

**Reply by Scialla** [reply]
Specifico meglio, non servono Tado Thread Border Routers

> You will need to add your tado° X room control devices one by one to the Alexa app, but you don’t need to add any tado° Thread border routers (e.g. Bridge X, Heat Pump Optimizer X, Wireless Receiver X).

**Reply by KC-inDomus** [reply]
nemmeno. il thread border router serve eccome. SE hai **echo 4gen**, allora appunto fa lui da border router (senza bisogno di prendere quello tado, appunto).

**Reply by Scialla** [reply]
E infatti io avevo scritto che serve un router compatibile matter e non serve un Tado thread router, mica me le sono inventate le ho riportate dal sito di Tado di cui ho riportato il link (santo Google)
Se clicchi sul link dice proprio così...

**Reply by KC-inDomus** [reply]
scialla scusa questa frase "Devi avere un Matter-compatible Alexa controller (non serve Thread)" è sbagliata, poi mettila come credi

**Reply by Scialla** [reply]
_ In order to control tado° X devices with Alexa, you will need a Matter-compatible Alexa controller (i.e. certain models of Echo and eero)_.

Riportato sul link di supporto (se avessi cliccato)

Per il "non serve thread" ho poi, giustamente, specificato meglio riportando la frase corretta del supporto

Lo so che tu vuoi evidenziare che sono dispositivi che funzionano con thread, ma era implicito nella precisazione che non servono TADO thread border router

**Reply by KC-inDomus** [reply]
infatti quest'altra frase

_In order to control tado° X devices with Alexa, you will need a Matter-compatible Alexa controller (i.e. certain models of Echo and eero)._

è altrettanto una cazzata: non è perché tado scrive una cosa, automaticamente diventa vangelo scia'...

**Reply by Scialla** [reply]
E vabbè, ma se il supporto ufficiale dice così.... pigliatela con loro! 😅

**Reply by danieva79** [reply]
Grazie a tutti. Nello specifico
Echo Show 8 (3ª gen., modello 2023)
Echo Dot (5ª generazione, modello 2022)

Potrebbero fare al caso mio? Ossia evitarmi di dover comprare il bridge X?

Una domanda, con "Home Assistant" cosa potrei farei in più per una "casa vacanze"? Mi rendo conto che la domanda è molto "aperta" ma è difficile trovare fonti di informazioni e raramente così disponibili. Grazie.

**Reply by Scialla** [reply]
Echo Dot no, lo Show 8 è compatibile Thread e Matter quindi si. 

Cosa potresti fare con Homeassistant te ne accorgerai da solo quando cercherai di fare qualcosa con le routine Alexa e non riuscirai a farlo.. con HA il limite è la fantasia

Le routine ti permettono di fare automazioni basilari che, prima o poi, ti staranno "strette"

**Reply by danieva79** [reply]
Così mi incuriosisci :-) 
Se posso, vale la pena di installare Homeassistant per una casa che non sia quella in cui si vive e per le automazioni (almeno, quelle, all'inizio) che ho in mente? In termini di complessità di programmazione ed interazione con Tado, Sensori Philips Hue e poco altro immagino non ci siano problemi. Grazie mille

**Reply by danieva79** [reply]
Grazie Scialla. Mi sembra però troppo complesso come "Inizio". Echo Show 8 può andare per i "sempliciotti" come me :-) o suggerite di orientarmi su altro (google è meglio?)? Grazie ancora
